src/gt4sd/frameworks/enzeptional/processing.py

```diff
@@ -116,6 +128,89 @@ def __call__(self, samples: List[str]) -> np.ndarray:
         )
 
 
+class ProtTransXL(StringEmbedding):
+    """Embed a string representation of a molecule using a ProtTrans_XLnet model"""
```

Comment:
are we sure the docstring and the class name are correct? to me it looks like the model is T5.

Reply:
but maybe it's a generic one